controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -200,9 +201,15 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	default:
 		switch platform {
 		case cluster.SelfManagedRhods:
-			err := r.createUserGroup(ctx, instance, "rhods-admins")
-			if err != nil {
-				return reconcile.Result{}, err
+			// Check if user opted for disabling creating user groups
+			disableUserGroup, exist := os.LookupEnv("DISABLE_USERGROUP")
```

Comment:
Ideally the env var name should be defined as a constant

Reply:
+1 @zdtsw Can we update the name to be platform agnostic?